File: platform/src/java/com/xcase/rest/generator/IAPIProxySettingsEndpoint.java
package com.xcase.rest.generator;

import java.lang.invoke.MethodHandles;
import java.util.*;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public interface IAPIProxySettingsEndpoint {
    boolean getAppendAsyncToMethodName();

    String getBasePath();

    String getBaseProxyClass();

    String getHost() throws Exception;

    String getId();

    String getNamespace();

    boolean getParseOperationIdForProxyName();

    String getProxyConstructorSuffix();

    String getUrl();
}

File: dotnetcore/XCaseServiceClient/XCase.ProxyGenerator/IAPIProxySettingsEndpoint.cs
namespace XCase.ProxyGenerator
{
    public interface IAPIProxySettingsEndpoint
    {
        public string GetAccept();

        bool GetAppendAsyncToMethodName();

        string GetBasePath();

        string GetBaseProxyClass();

        string GetHost();

        string GetId();

        string GetNamespace();

        bool GetParseOperationIdForProxyName();

        string GetProxyConstructorSuffix();

        public string GetTokenName();

        string GetUrl();
    }
}
